### ISSUE TITLE
findIssue properties default?

### ISSUE THREAD

**Issue by beb-bang** [jira-node/node-jira-client#187]
Hi - I'm having some trouble with the basic findIssue sample provided in the readme and was hoping you could help.

When I run this:
```
var issueNumber = 'CC-1111';
jira.findIssue(issueNumber)
  .then(issue => {
    console.log(issue);
  })
  .catch(err => {
    console.error(err);
  });

```
I receive an "500 Internal Service Error" from JIRA. In my experience (based almost entirely on building the same type of library for PowerShell), that's a problem with the request (query string on get or json body on post). 

This is the uri returned in err:
`https://jira.company.com/rest/api/2/issue/CC-1111?expand=&fields=*all&properties=&fieldsByKeys=false
`

When I plug that into something like Insomnia or Postman, I get the same 500 error. 
However, when I add a '*' to the properties field it works great (in a REST browser).

So, back in Node, when I call findIssue like so (forcing some '*' values) it also works great.

```
var issueNumber = 'CC-1111';
jira.findIssue(issueNumber,'*','*','*','*')
  .then(issue => {
    console.log(issue);
  })
  .catch(err => {
    console.error(err);
  });
```

Am I doing something wrong? Does the properties field need a default value? Or perhaps the JIRA version I'm running (7.3.1) is expecting something a little different? 

Any advice is greatly appreciated.

Thx!
Bryan

**Comment by KanoYugoro** [comment]
The issue I believe is the parts of the URL that don't have anything assigned to them by default, such as the `expand` or `properties` in your example URL.  It is also possible that the specific version of jira that you're on has an issue.

**Comment by sandyleo26** [comment]
I have the same issue. Thought it was usage error.

**Comment by sgtoj** [comment]
I am running in to the same issue with the same solution as @sandyleo26  and @beb-bang. I think this issue started with NodeJS v8.0. I am using this module in production with a NodeJS v.7.x runtime without issues.

**Comment by timcrider** [comment]
This issue does not exist in 6.3.0. I reverted from 6.4.0 to 6.3.0 and the issue was resolved.

**Comment by metaodi** [comment]
Any comments on #190?

**Comment by metaodi** [comment]
Fixed in #202

**Comment by pioug** [comment]
Closing since all looks fine now 🙏
